docs/web_reference.rst

```diff
@@ -2793,6 +2794,10 @@ Utilities
                                 timeout but closes a server in a few
                                 milliseconds.
 
+   :param float keepalive_timeout: a delay before a TCP connection is
```

Comment:
The argument name is ambiguous on this level because `run_app()` doesn't suggest that it has anything to do with TCP. I'd probably call the user-facing interface `tcp_keepalive_timeout`.

Reply:
Well, the function already leaks this with the `port` parameter. I'm not even sure we can say that HTTP keepalive is restricted to TCP - couldn't any transport layer protocol with a handshake be used here?

Reply:
My point is that HTTP keep-alive and TCP keep-alive are different things. And so it's not clear which one it is just based on the name.

Reply:
I'm trying to add support for changing HTTP keepalive timeout, so prefixing it with `tcp_` would only make things worse.

Reply:
Yes, the value is for http keepalive, not tcp keepalive. Let's keep it